doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
+
+The IoT Edge Hub module exposes a number of metrics in the Prometheus exposition format that provide insights into its operational state.
+
+### How to enable
+
+As of release 1.0.9, metrics are exposed as an experimental feature available at http port **9600** of the Edge Hub module. To enable, the following environment variables should be set for the module (make note of the double underscores):
```

Comment:
The port and address is configurable. Maybe we should add that as well. 
metrics:listener:port (default 9600)
metrics:listener:suffix (default metrics)
Also, should we put this flag under experimentalFeaures as well? (Currently only the enabling of metrics is under experimental)

Reply:
Yes, looks like we should put in under experimental. I can update the docs after code changes for that are made.